packages/gatsby-theme/gatsby-theme-eject.js

```diff
@@ -2,7 +2,7 @@
 "use strict";
 const fs = require("fs");
 const path = require("path");
-const glob = require("glob");
+const glob = require("globby");
```

Comment:
Why switch here but not on different places?

Reply:
gatsby-theme was using both glob and globby. Including two different glob libraries in one package seemed redundant to me, so I just chose one over the other. glob works fine everywhere else, so I didn't change it across the whole monorepo